<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>445</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <property name="styleSheet" >
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="cursor" >
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QSplitter" name="splitter" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible" >
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="layoutWidget" >
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QListView" name="filterList" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="filterOptions" >
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="imageViewFrame" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QSpinBox" name="pageSpinBox" />
      </item>
      <item>
       <widget class="QSlider" name="pageSlider" >
        <property name="tracking" >
         <bool>true</bool>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>613</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDebug" >
    <property name="title" >
     <string>Tools</string>
    </property>
    <addaction name="actionDebug" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionSaveProject" />
    <addaction name="actionSaveProjectAs" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuDebug" />
  </widget>
  <action name="actionDebug" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Debug</string>
   </property>
  </action>
  <action name="actionSaveProject" >
   <property name="text" >
    <string>Save Project</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveProjectAs" >
   <property name="text" >
    <string>Save Project As ...</string>
   </property>
  </action>
  <action name="actionNextPageOnPgDown" >
   <property name="text" >
    <string>Next Page</string>
   </property>
   <property name="shortcut" >
    <string>PgDown</string>
   </property>
   <property name="autoRepeat" >
    <bool>false</bool>
   </property>
  </action>
  <action name="actionPrevPageOnPgUp" >
   <property name="text" >
    <string>Previous Page</string>
   </property>
   <property name="shortcut" >
    <string>PgUp</string>
   </property>
   <property name="autoRepeat" >
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pageSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>pageSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>263</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel" >
     <x>55</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pageSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pageSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>55</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel" >
     <x>301</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNextPageOnPgDown</sender>
   <signal>triggered(bool)</signal>
   <receiver>pageSpinBox</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>37</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPrevPageOnPgUp</sender>
   <signal>triggered(bool)</signal>
   <receiver>pageSpinBox</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>37</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
